app/assets/stylesheets/base/_buttons.scss

```diff
@@ -33,3 +33,9 @@
     }
   }
 }
+
+#get_location{
```

Comment:
Could above with a style to differentiate it from the submit button be clearer?

![screen shot 2016-05-27 at 6 14 11 pm](https://cloud.githubusercontent.com/assets/1239550/15602175/fce4c02a-2436-11e6-81f3-bcc73739d979.png)